roles/confluent.control_center/tasks/main.yml

```diff
@@ -61,6 +61,15 @@
     group: "{{control_center_group}}"
     recurse: true
 
+# Archive File deployments need to create SystemD directory on Debian
+- name: Create Zookeeper Service path
+  ansible.builtin.file:
+    path: "/usr/lib/systemd/system"
```

Comment:
I think it is considered best practice to set owner, group and permissions.

Reply:
It adopts the OS defaults, and because this is a system directory it defaults to the following which is what is desired: 

drwxr-xr-x 2 root root 4096 Apr 20 09:26 system

So we can specify the owner group and permissions, but it's not required in this instance.